src/main/webapp/content/scss/themes/_default-variables.scss

```diff
@@ -2,7 +2,7 @@
 // Mostly default values, including them here for completeness and comparison
 
 $white: #fff;
-$gray-100: #f8f9fa;
+$gray-100: #fbfdff;
```

Comment:
Do I see correctly that this is a global variable? What is the reason behind this change and did you discuss it? It might affect several places within Artemis.

Reply:
I double checked this and this is by mistake here, it should not be changed.